pyriemann/classification.py

```diff
@@ -903,4 +902,106 @@ def predict_proba(self, X):
         prob : ndarray, shape (n_matrices, n_classes)
             Probabilities for each class.
         """
-        return softmax(-self._predict_distances(X)**2)
+        return softmax(-self._predict_distances(X) ** 2)
+
+
+def class_distinctiveness(X, y, return_num_denom=False):
+    r"""Measure class separability between classes on the manifold.
+
+    For two class problem, the class distinctiveness between class A
+    and B on the manifold is quantified as:
+
+    .. math::
+        \mathrm{classDis}(A, B) = \frac{\delta_{R}
+                       \left(\bar{C}^{A}, \bar{C}^{B}\right)}
+                       {\frac{1}{2}
+                       \left( \sigma_{C^{A}} + \sigma_{C^{B}}
+                       \right)}
+
+    where :math:`\bar{C}^{K}` and :math:`\sigma_{C^{K}}` are
+    respectively the mean and the mean absolute deviation of the
```

Comment:
I kept the mean for clarity because the absolute deviation can be based on the mean or the median. The original equation uses the mean absolute deviation.